Projects/Features/SearchFeature/Sources/View/BeforeSearchSectionHeaderView.swift

```diff
@@ -0,0 +1,67 @@
+import DesignSystem
+import SnapKit
+import Then
+import UIKit
+
+protocol BeforeSearchSectionHeaderViewDelegate: AnyObject {
+    func tap(_ section: Int?) -> Void
+}
+
+class BeforeSearchSectionHeaderView:
+    UICollectionReusableView {
+    static let kind = "before-search-section-header"
+
+    weak var delegate: BeforeSearchSectionHeaderViewDelegate?
+
+    var section: Int? = nil
+
+    var label: UILabel = UILabel().then {
+        $0.font = DesignSystemFontFamily.Pretendard.medium.font(size: 16)
+
+        $0.textColor = DesignSystemAsset.BlueGrayColor.blueGray900.color
+    }
+
+    var button: UIButton = UIButton().then {
+        $0.setTitle("전체보기", for: .normal)
+
+        $0.setTitleColor(DesignSystemAsset.BlueGrayColor.blueGray900.color, for: .normal)
+
+        $0.titleLabel?.font = DesignSystemFontFamily.Pretendard.medium.font(size: 14)
+    }
```

Comment:
```suggestion
    private let label: UILabel = UILabel().then {
        $0.font = DesignSystemFontFamily.Pretendard.medium.font(size: 16)
        $0.textColor = DesignSystemAsset.BlueGrayColor.blueGray900.color
    }

    private let button: UIButton = UIButton().then {
        $0.setTitle("전체보기", for: .normal)
        $0.setTitleColor(DesignSystemAsset.BlueGrayColor.blueGray900.color, for: .normal)
        $0.titleLabel?.font = DesignSystemFontFamily.Pretendard.medium.font(size: 14)
    }
```
private 붙여주시면 좋을거같아요. 외부 접근이 가능해서 코드를 생각해야하는 범위가 너무 넓어져서요

Reply:
적용했습니다.